### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by isantospardo** [promitor/charts#87] [Q&A] [answered]
Would it be possile to create a new release with latest changes?

**Comment by tomkerkhove** [comment, marked as answer]
It was blocked because of https://github.com/promitor/charts/issues/80 but is now available on https://github.com/promitor/charts/releases/tag/Scraper-v2.6.3

**Comment by isantospardo** [comment]
Could we also bump Promitor Resource Discovery Chart to v0.6.1?

**Reply by tomkerkhove** [reply]
There will be a full release tomorrow, but I don't think resource discovery has changed.

**Reply by isantospardo** [reply]
Thanks, we also added https://github.com/promitor/charts/commit/9397585847b63e3cc215891c61ba19c6a9db4595#diff-11b72e0226516b5fb88e86e821f557e59806bc54b2683872ed5284145e151f34, right?

**Reply by tomkerkhove** [reply]
Correct, sorry!
